source/pages/Tokens/CustomToken.tsx

```diff
@@ -47,13 +49,17 @@ export const CustomToken = () => {
         web3Provider
       );
 
+      const balance = ethers.utils.formatEther(metadata.balance);
```

Comment:
We need to format the token by the amount of decimals the token has, only for blockchain native tokens it should be used format either function

Reply:
Done. Here is the comparation between metamask and pali: 
![Screen Shot 2022-11-15 at 12 07 29](https://user-images.githubusercontent.com/72104971/201954790-40d971cb-f445-4db1-877e-00110c064c9a.png)
![Screen Shot 2022-11-15 at 12 08 54](https://user-images.githubusercontent.com/72104971/201954814-c55eb568-db28-4032-a1ea-20cff1007f32.png)